### DISCUSSION TITLE
Python >= 2.7

### DISCUSSION THREAD

**Discussion by DimitriPapadopoulos** [brainvisa/brainvisa.github.io#146] [Q&A] [answered]
Do the current [Prerequisites](https://brainvisa.info/web/download.html#prerequisites) still include Python 2.7?

> For Linux, two software must be installed : Python (version >= 2.7) and Singularity (version > 3.6).

**Comment by denisri** [comment, marked as answer]
Not exactly actually. Releases 5.0.x could use either python 2 or 3, and 5.1.x require python3. We must clarify it in the doc, thanks.
